server/src/server.ts

```diff
@@ -71,6 +73,8 @@ db.once('open', () => console.log('Connected to database'));
 app.use('/geo', geoRoutes);
 app.use('/auth', authRoutes);
 app.use('/packages', packageRoutes);
+app.use('/histories', historyRoutes);
```

Comment:
```suggestion
app.use('/history', historyRoutes);
```
I think better suited

Reply:
Routes should be in plural form